server/src/controller/card/card.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { controller, httpGet, httpPost, httpDelete } from 'inversify-express-utils';
+import { Request, Response } from 'express';
+import { inject } from 'inversify';
+import { CardRepository } from './../../service/card/card.repository';
+
+@controller('/api')
+export class CardController {
+    public constructor(
+        @inject(CardRepository) private cardRepository: CardRepository
+    ) { }
+
+    @httpGet('/get-cards/')
+    public async getSkills(request: Request, response: Response){ 
+        try {
+            return await this.cardRepository.getCards();
```

Comment:
you can remove await, in async function all return promises will automatically waitable